api/src/main/java/org/openmrs/module/xdssender/api/scheduler/PullNotificationsTask.java

```diff
@@ -23,18 +27,33 @@ public class PullNotificationsTask extends AbstractTask {
 	public void execute() {
 		LOGGER.info("Executing " + TASK_NAME);
 		HL7Service hl7Service = Context.getHL7Service();
+		Date newDate = new Date();
+		Boolean success = true;
```

Comment:
Always use primitives unless you need `null`;

```suggestion
		boolean success = true;
```